kite-app-parent/cdh5/pom.xml

```diff
@@ -67,15 +67,20 @@
   </scm>
 
   <properties>
+    <!-- Scala versions -->
+    <scala.base.version>2.10</scala.base.version>
+    <scala.version>2.10.5</scala.version>
+    <scalatest.version>2.2.4</scalatest.version>
+
     <!-- dependency version properties -->
     <kite.version>1.0.1-SNAPSHOT</kite.version>
-    <kite.avro.version>1.7.5-cdh5.1.0</kite.avro.version>
-    <kite.crunch.version>0.10.0-cdh5.1.0</kite.crunch.version>
+    <kite.avro.version>1.7.6-cdh5.2.0</kite.avro.version>
+    <kite.crunch.version>0.11.0-cdh5.2.0</kite.crunch.version>
     <kite.findbugs.version>2.0.1</kite.findbugs.version>
-    <kite.hive.version>0.12.0-cdh5.1.0</kite.hive.version>
-    <kite.oozie.version>4.0.0-cdh5.1.0</kite.oozie.version>
-    <kite.spark.version>1.0.0-cdh5.1.0</kite.spark.version>
-    <kite.parquet.version>1.2.5-cdh5.1.0</kite.parquet.version>
+    <kite.hive.version>0.13.1-cdh5.2.0</kite.hive.version>
+    <kite.oozie.version>4.0.0-cdh5.2.0</kite.oozie.version>
+    <kite.spark.version>1.3.0-cdh5.4.0</kite.spark.version>
+    <kite.parquet.version>2.1.0-cdh5.2.0</kite.parquet.version>
```

Comment:
Got it. I would update to the latest. Is there any downside to doing that, @rdblue?